src/Place/ReadModel/Enum/EventDescription.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\Enum;
+
+/**
+ * Class EventDescription
+ * @package CultuurNet\UDB3\Place\ReadModel\Enum
+ *
+ * @TODO: FIX BEFORE MERGE - use real names for description
+ */
+class EventDescription
+{
+    const CREATED = 'Created';
+    const DELETED = 'Deleted';
+    const LABEL_ADDED = 'label-added';
+    const LABEL_REMOVED = 'label-removed';
+    const DESCRIPTION_TRANSLATED = 'description-translated';
+    const TITLE_TRANSLATED = 'title-translated';
+    const PLACE_IMPORTED_FROM_UDB2 = 'place-imported-from-udb2';
+    const PLACE_UPDATED_FROM_UDB2 = 'place-updated-from-udb2';
```

Comment:
This class is not used anymore right?

Reply:
Yes, but just check if the description for a deleted event is ok now